src/app/[locale]/(default)/events/page.tsx

```diff
@@ -21,5 +26,84 @@ export default async function EventsPage({
 }) {
   const { locale } = await params;
   setRequestLocale(locale);
-  return <div>This should be events page</div>;
+  const t = await getTranslations('events');
+  const tUi = await getTranslations('ui');
+
+  const translations = {
+    internal: tUi('internal'),
+    startsAt: t('startsAt'),
+    startedAt: t('startedAt'),
+    endsAt: t('endsAt'),
+    endedAt: t('endedAt'),
+  };
+
+  return (
+    <>
+      <h1 className='my-4'>{t('title')}</h1>
+      <Link href='#active'>
+        <h2 className='my-2' id='active'>
```

Comment:
```suggestion
        <h2 className='pt-4 mb-4' id='active'>
```
Should have consistent spacing